lib/index_group.rb

```diff
@@ -11,9 +11,9 @@ class IndexGroup
     def initialize(base_uri, name, schema, search_config)
       @base_uri = base_uri
       @client = Services::elasticsearch(hosts: base_uri)
-      # Index creation can take longer than other requests, so create a separate
+      # Index creation/deletion can take longer than other requests, so create a separate
       # client with a longer timeout
-      @index_creation_client = Services::elasticsearch(hosts: base_uri, timeout: 10)
+      @long_timeout_client = Services::elasticsearch(hosts: base_uri, timeout: 30)
```

Comment:
Had to extend this as the deletion calls could time out even with a 10 second leeway.